CHANGELOG.md

```diff
@@ -1,5 +1,104 @@
+* support trailing slashes in listener URLs ([#5076](https://github.com/confluentinc/ksql/pull/5076)) ([e9e0431](https://github.com/confluentinc/ksql/commit/e9e0431f6eafd7ef18dd2cb9e17730ec3ba1e226))
+* backport fixes from query close ([#4819](https://github.com/confluentinc/ksql/pull/4819)) ([76f960b](https://github.com/confluentinc/ksql/commit/76f960bc5702d3995d30ba538956813728160862)), closes [#4643](https://github.com/confluentinc/ksql/issues/4643) [#4658](https://github.com/confluentinc/ksql/issues/4658)
+* logic when closing ksqlEngine fixed ([#4917](https://github.com/confluentinc/ksql/pull/4917)) ([a217eb9](https://github.com/confluentinc/ksql/commit/a217eb9a8a80b38c970021faad76a4b9fcade609))
+* don't cleanup topics on engine close ([#4658](https://github.com/confluentinc/ksql/pull/4658)) ([#4857](https://github.com/confluentinc/ksql/issues/4857)) ([21e4751](https://github.com/confluentinc/ksql/commit/21e47519ffd8df8cfee083e559ac28f736f058c1))
+
+
+
+### chore
+
+* add GROUP BY support for any key names ([#4899](https://github.com/confluentinc/ksql/pull/4899)) ([e7cbdfc](https://github.com/confluentinc/ksql/commit/e7cbdfcc8c9853e2ae6dfcaf670e04f07ccd5444)), closes [#4898](https://github.com/confluentinc/ksql/issues/4898)
+
+
+
+### BREAKING CHANGES
+
+* Select star, i.e. `select *`, no longer expands to include `ROWTIME` column(s). Instead, `ROWTIME` is only included in the results of queries if explicitly included in the projection, e.g. `select rowtime, *`.
+This only affects new statements. Any view previously created via a `CREATE STREAM AS SELECT` or `CREATE TABLE AS SELECT` statement is unaffected.
+Co-authored-by: Andy Coates <big-andy-coates@users.noreply.github.com>
+* This commit changes the system generated column name for any columns in projections that are struct field dereferences. Previously, the full path was used when generating the name, now only the final field name is used. For example, `SELECT someStruct->someField, ...` would previously of generated a column name of `SOMESTRUCT__SOMEFIELD`, and will now generate a name of `SOMEFIELD`. Generated column names may have a numeric appended to the end to ensure uniqueness, for example `SOMEFIELD_2`.
+Note: it is recommended that you do not rely on system generated column names for production systems as there naming may change between releases. Providing an explict alias ensures consistent naming across releases. For example, `SELECT someStruct->someField AS someField`.
+Backwards compatibility: existing running queries will not be affected by this change: they will continue to run with the same column names. Any statements executed after the upgrade will use the new names where no explicit alias is provided. Add explicit aliases to your statements if you require the old names, for example: `SELECT someStruct->someField AS SOMESTRUCT__SOMEFIELD, ...`
+Co-authored-by: Andy Coates <big-andy-coates@users.noreply.github.com>
```

Comment:
Why are names and email addresses in this changelog? Doesn't look right?

Reply:
The changelog generator blindly copy-pastes whatever is in the commit message. This line was part of the commit message, and was picked up as a result: https://github.com/confluentinc/ksql/commit/2002458f6b668b7b17b05b192a7cc3fab0a23b20

Reply:
I just removed them.